build/azure-pipelines/darwin/createDrop.sh

```diff
@@ -6,7 +6,9 @@ REPO="$(pwd)"
 mkdir -p $REPO/.build/darwin/{archive,server}
 
 # remove pkg from archive
-zip -d $REPO/.build/darwin/archive/azuredatastudio-darwin.zip "*.pkg"
+if [[ "$signed" == "true" ]]; then
```

Comment:
Not sure if this matters on linux/darwin but ado transforms the env vars and makes them all upper case. so if the var in the pipeline is `signed` the env var will be `SIGNED`.